Dockerfile

```diff
@@ -0,0 +1 @@
+images/cluster-ingress-operator/Dockerfile
```

Comment:
Would prefer to avoid symlinks... because other out-of-tree configurations need to reference our Dockerfiles, I'd rather encourage pointing to physical files since symlinks can be problematic depending on the context.

Reply:
the problem is when making the copy of the source to the image filesystem while building since dockerfiles don't allow access to higher level directories. It might then be better to create hack/build-image.sh script